accounts/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import SignUpView
+
+urlpatterns = [
+    path("signup/", SignUpView.as_view(), name="signup"),
```

Comment:
What about url for the logout?

Reply:
It is available from this : 
path('accounts/', include("django.contrib.auth.urls"))
In xpense.urls

It is available at "accounts/logout"

Reply:
thanks
you can mark this conflict as resolved :)